src/lib/components/submit/SubmitWrite.svelte

```diff
@@ -1,28 +1,20 @@
 <script lang="ts">
-	import type { ProposalContent } from '$lib/types/juno';
-	import SubmitContinue from '$lib/components/submit/SubmitContinue.svelte';
-	import { createEventDispatcher, onMount } from 'svelte';
-	import SubmitWriteEdit from '$lib/components/submit/SubmitWriteEdit.svelte';
-	import type { ProposalEditableMetadata } from '$lib/types/juno';
-	import { getEditable } from '$lib/services/idb.services';
+    import SubmitContinue from '$lib/components/submit/SubmitContinue.svelte';
+    import { createEventDispatcher} from 'svelte';
+	import SubmitMotion from '$lib/components/submit/SubmitMotion.svelte';
+	import SubmitAddNodeProvider from '$lib/components/submit/SubmitAddNodeProvider.svelte';
 
-	let metadata: ProposalEditableMetadata | undefined;
-	let content: ProposalContent | undefined;
+    export let proposalType: string | undefined;
 
-	onMount(async () => ([metadata, content] = await getEditable()));
-
-	const dispatch = createEventDispatcher();
-	const next = () => dispatch('pnwrkNext');
+    const dispatch = createEventDispatcher();
+    const next = () => dispatch('pnwrkNext');
 </script>
 
-<h1 class="font-bold capitalize text-4xl md:text-6xl mb-12">Craft Your Proposal</h1>
-
-<h2 class="text-2xl mb-6">
-	To submit a proposal, you need both a summary (explaining what it's all about) and motion text
-	(the effective motion for voting), along with a title and a URL pointing to your community
-	discussion.
-</h2>
-
-<SubmitWriteEdit {metadata} {content} />
+{#if proposalType === 'Motion'}
```

Comment:
should we display an error (a text) for the edge case where `proposalType` would not be set? there is a change here that the user ends on an empty screen given that there is not final `:else`. WDYT?

Reply:
In this [comment](https://github.com/peterpeterparker/proposals.network/pull/8#discussion_r1488317003), I created a TODO so that a user would not be able to select the undefined state. If this is done then an error message won't be necessary.

Just in general, the least we can write error messages the better. If we can make the UI so that the user cannot make these errors, that would be ideal imo.

Reply:
Thanks for the exaplanation. Ok let's keep it as it and revisit it in the future if necessary.